target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8065-r7800.dts

```diff
@@ -45,6 +45,13 @@
 			pinctrl-0 = <&mdio0_pins &rgmii2_pins>;
 			pinctrl-names = "default";
 
```

Comment:
```pinctrl-0 = <&mdio0_pins &rgmii2_pins>;``` as child of ```pinmux@800000``` looks rather wrong.

> It's used this way in chromium and Qualcomm sdk repositories

Our reference is the linux kernel code and not some repo of a vendor with hackish code. It is no secret that vendor code doesn't have the same level of quality as kernel code.

In my opinion this should be fixed. But since @heinzek did not touched that code I don't insist that it is done with this PR.